fuel-relayer/src/config.rs

```diff
@@ -1,40 +1,61 @@
 use ethers_core::types::{H160, H256};
+use fuel_core_interfaces::model::DaBlockHeight;
 use std::{str::FromStr, time::Duration};
 
 lazy_static::lazy_static! {
-    pub static ref ETH_ASSET_DEPOSIT : H256 = H256::from_str("0x0000000000000000000000000000000000000000000000000000000000000001").unwrap();
-    pub static ref ETH_VALIDATOR_DEPOSIT : H256 = H256::from_str("0x0000000000000000000000000000000000000000000000000000000000000002").unwrap();
-    pub static ref ETH_VALIDATOR_WITHDRAWAL : H256 = H256::from_str("0x0000000000000000000000000000000000000000000000000000000000000003").unwrap();
-    pub static ref ETH_FUEL_BLOCK_COMMITED : H256 = H256::from_str("0x0000000000000000000000000000000000000000000000000000000000000004").unwrap();
+    /// Hash over solidity function: `DepositMade(uint32,address,address,uint8,uint256,uint256)`
```

Comment:
I think we should file a tech debt issue to replace this with the actual hashing code, so that it's easier to update these in the future.

Reply:
here it is: https://github.com/FuelLabs/fuel-core/pull/345/commits/8a1fbbd182ced087a2d2142f91013f3fe112d1c7